edge-apps/countdown-timer/screenly.yml

```
@@ -40,13 +40,13 @@ settings:
   tag_manager_id:
     type: string
     default_value: GTM-P98SPZ9Z
-    title: tag_manager_id
+    title: Tag Manager ID
     optional: true
     help_text: Specify a Google Tag Manager ID
     is_global: true
   target_timestamp:
     type: string
     default_value: 2025-01-01T00:00:00
-    title: target_timestamp
+    title: Target Countdown Date & Time
     optional: false
-    help_text: Countdown timer value (Format YYYY-MM-DDTHH:MM:SS).
+    help_text: Target Countdown Date & Time (Format YYYY-MM-DDTHH:MM:SS).
```

```suggestion
    help_text: "Format: YYYY-MM-DDTHH:MM:SS"
```

Just make it useful without extra repeating text